app/routes/login.js

```diff
@@ -0,0 +1,13 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
```

Comment:
I think you can directly import service?

Reply:
yes, i just followed it as it was in other components, but i'll change it.